tasks/install.yml

```
@@ -122,17 +126,15 @@
     nonsensitive_stdout.skipped is not defined
     or not nonsensitive_stdout.skipped | bool
 
+- name: Install | Revert State Upon Error
```

Given our other discussion, just rename this and we can merge!

```suggestion
- name: Install | Clear State Upon Error
```